uint8 isJoint
uint8 cmd
---
int32 result
uint64 queuedCmdIndex
